emitter.py

```
@@ -126,6 +126,7 @@ def http_emitter(message, log, agentConfig, endpoint):
             log.error("Received status code: {0}".format(r.status_code))
         except Exception:
             pass
+        raise Exception("Posting payload failed")
```

Why do we need to raise an exception here? I'm wondering, since we're changing core code here, it may affect other parts of the agent in an unexpected way.

---

this was needed to find out whether the emitter successfully posted data